contents/docs/integrate/_snippets/install-elixir.mdx

```diff
@@ -1,4 +1,4 @@
-> This library was built by the community and is not maintained by the PostHog core team.
+> This library was originally built by the community but it's being maintained by the PostHog core team since v1.0.0. Thank you to [Nick Kezhaya](https://github.com/nkezhaya) for building it originally.
```

Comment:
I don't think we should have originally twice in two sentences.
```suggestion
> This library was built by the community but it's being maintained by the PostHog core team since v1.0.0. Thank you to [Nick Kezhaya](https://github.com/nkezhaya) for building it originally.
```